src/resources/js/tribe-events-ajax-calendar.js

```diff
@@ -180,16 +180,21 @@
 
 		function tribe_mobile_month_setup() {
 
-			var $today          = $wrapper.find( '.tribe-events-present' );
+			var $active_day       = $wrapper.find( '.mobile-active' );
```

Comment:
Lets use the `$activeDay` since the new Standard is camelCase for JS.

Reply:
Yep, changed most of the others as well 😉